Lib/test/test_tokenize.py

```diff
@@ -1873,6 +1878,33 @@ def test_indentation_semantics_retained(self):
         self.check_roundtrip(code)
 
 
+class InvalidPythonTests(TestCase):
+    def test_number_followed_by_name(self):
+        # See issue #gh-105549
+        source = "xsin(x)"
```

Comment:
There is no number before the name.  This currently parses in main (just recompiled) as expected below *without* this patch.  Did you mean something like "2sin(x)", which currently raises TokenError("('invalid decimal literal', (1, 1))")?

Reply:
> Did you mean something like "2sin(x)", which currently raises TokenError("('invalid decimal literal', (1, 1))")?

Yeah, thanks for the catch!